src/reporters/base-reporter.js

```diff
@@ -120,6 +120,7 @@ export default class BaseReporter {
     try {
       standardInput = process.stdin;
```

Comment:
could you please add a text comment why we do the trick?
Going forward this file may be refactored multiple times and we would loose the original reason for try/catch.